torch/csrc/jit/ir/ir.h

```diff
@@ -1512,5 +1518,68 @@ struct OperatorSet {
   std::unordered_map<Symbol, std::vector<std::shared_ptr<Operator>>> ops;
 };
 
+template <typename T>
+struct OperatorMap {
```

Comment:
Nit: maybe move more of the implementation to `ir.cpp` following along with convention for `OperatorSet`